pkgs/development/python-modules/typer/default.nix

```
@@ -18,14 +18,14 @@
 
 buildPythonPackage rec {
   pname = "typer";
-  version = "0.12.5";
+  version = "0.15.1";
   format = "pyproject";
 
   disabled = pythonOlder "3.6";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-9ZLwib7cyOwbl0El1khRApw7GvFF8ErKZNaUEPDJtyI=";
+    hash = "sha256-oFiMCn+mihl4oGmBhld3j4ar5v9epqv0cvlAoIv+Two=";
   };
 
   nativeBuildInputs = [ pdm-backend ];
```

```suggestion
  build-system = [ pdm-backend ];
```

and dependencies, please.